pyfr/solvers/baseadvecdiff/system.py

```diff
@@ -50,6 +68,12 @@ def deps(dk, *names): return self._kdeps(k, dk, *names)
         for l in k['mpiint/con_u']:
             g2.add(l, deps=deps(l, 'mpiint/scal_fpts_unpack'))
 
+        # Compute common entropy minima at MPI interfaces
+        if 'eles/local_entropy' in k:
```

Comment:
Is this explicit check needed?

Reply:
I believe so, mpicent operates on entmin_lhs which is None if entropy filtering isn't active

Reply:
Usually the way we handle this is by not defining the kernel in the interface side (see how artificial viscosity handles its analogous case).  Easier and more efficient to prune upstream.